### DISCUSSION TITLE
Add for jwtVerify option to throw an exception if exp date is past

### DISCUSSION THREAD

**Discussion by iamdeveloperidevelop** [panva/jose#445] [Ideas]
I'd love to send param to jose.jwtVerify to throw exception whatever is wrong with JWT. If it's invalid, expired, pick whatever other issue. Not to have validate expiry date myself

**Comment by panva** [comment]
What makes you think you need to check exp yourself? Both exp and nbf are checked out of the box, no option needed.

**Reply by iamdeveloperidevelop** [reply]
I actually have no idea.. My bad. I was too rushed to test it correctly. Indeed I get the erorr. Thanks
I'd love to close it but I don't see an option.
